packages/api/amplify_api/example/lib/graphql_api_view.dart

```diff
@@ -62,127 +69,549 @@ class _GraphQLApiViewState extends State<GraphQLApiView> {
+                        : null,
+                    style: ElevatedButton.styleFrom(
+                      backgroundColor: Colors.amber, // This is what you need!
+                    ),
+                    child: const Text('Comment'),
+                  ),
+                ),
+              ],
+            ),
+
+            /// Subscriptions
+            ///
+            const SizedBox(
+              width: double.infinity,
+              child: Text(
+                'Subscriptions',
+                textAlign: TextAlign.left,
+              ),
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                ElevatedButton(
+                  onPressed: widget.isAmplifyConfigured && _subscription == null
+                      ? subscribe
+                      : null,
+                  child: const Text('Blogs'),
+                ),
+                ElevatedButton(
+                  onPressed: widget.isAmplifyConfigured &&
+                          _subscriptionByID == null &&
+                          _blog != null
+                      ? subscribeByID
+                      : null,
+                  child: const Text(
+                    'Posts By BlogID',
+                    maxLines: 1,
+                  ),
+                ),
+                ElevatedButton(
+                  onPressed: _subscription != null
+                      ? () => setState(() {
+                            _unsubscribe?.call();
+                            _unsubscribe = null;
+                            _subscription = null;
+                          })
+                      : null,
+                  child: const Text('Unsubscribe'),
+                ),
+              ],
+            ),
+            const Padding(padding: EdgeInsets.all(10)),
+
+            const SizedBox(
+              width: double.infinity,
+              child: Text(
+                'Authorization Type',
+                textAlign: TextAlign.left,
+              ),
+            ),
+            Row(
+              children: <Widget>[
+                Radio<APIAuthorizationType>(
+                  value: APIAuthorizationType.apiKey,
+                  groupValue: _authorizationType,
+                  onChanged: (APIAuthorizationType? value) {
+                    setState(() {
+                      _authorizationType = value;
+                    });
+                  },
+                ),
+                const Text(
+                  'API Key',
+                ),
+                Radio<APIAuthorizationType>(
+                  value: APIAuthorizationType.iam,
+                  groupValue: _authorizationType,
+                  onChanged: (APIAuthorizationType? value) {
+                    setState(() {
+                      _authorizationType = value;
+                    });
+                  },
+                ),
+                const Text('iam'),
```

Comment:
These would be better grouped in a `ListTile`. Check out the Radio docs for an example: https://api.flutter.dev/flutter/material/Radio-class.html